System/Cryptography/Encryption.py

```diff
@@ -1,25 +1,37 @@
+###########################################################
+## This file is part of the BrainGenix Simulation System ##
+###########################################################
+
 from Cryptography.KeyUtils import GenKeys
 from Crypto.Util import number
 from Crypto.Hash import SHA256
 
+'''
+Name: Encryption
+Description: This file [fill this out here].

```

Comment:
@PRISHIta123 Are you able to write a short description of the encryption file here?

Reply:
Yes I will write that